scio-parquet/src/main/scala/com/spotify/scio/parquet/avro/ParquetAvroIO.scala

```diff
@@ -219,13 +212,27 @@ object ParquetAvroIO {
       }
     }
 
+    def setReadSchemas[A, T](params: ParquetAvroIO.ReadParam[A, T]): Unit = {
```

Comment:
`ReadParam` is already parameterized with `[A, T]`. I don't think we want type shadowing here

Reply:
Removing this function